packages/graphql-server/src/index.ts

```diff
@@ -1,11 +1,16 @@
-// In transition can keep Apollo Server error compatibility
-// export * from './errors'
+/**
+ * @deprecated
```

Comment:
@dac09 I am not sure how best the make everything here deprecated -- not how to test that it is,